### ISSUE TITLE
Wallpapers don't apply on startup

### ISSUE THREAD

**Issue by 0xk1f0** [danyspin97/wpaperd#59]
Hi, while adapting my tool to the newest `1.0` release I noticed that `wpaperd` does not apply the specified wallpapers in the config when first starting up. I need to manually issue a `wpaperctl` command to get it to respond, which led to a pretty ugly workaround in my tool. I don't know if this is intended?

<details>
<summary><code>config.toml</code></summary>

```toml
[default]
mode = "center"

[any]
path = "/home/k1f0/Documents/wallpapers/wallhaven-gpl8wl.jpg"

[DP-1]
path = "/home/k1f0/.cache/rwpspread/rwps_68c97b74c9e3fa7d_DP-1.png"

[DP-2]
path = "/home/k1f0/.cache/rwpspread/rwps_68c97b74c9e3fa7d_DP-2.png"
```
</details>

## Steps to reproduce (Arch Linux)

1. Build `wpaperd` from latest master
2. Define a valid configuration as mentioned [here](https://github.com/danyspin97/wpaperd?tab=readme-ov-file#wallpaper-configuration)
3. Start using `wpaperd` or `wpaperd -d`
4. No Wallpapers get applied

## Current Workaround

1. Start `wpaperd`
2. After startup, manually issue either `wpaperctl all-wallpapers` or `wpaperctl next-wallpaper`
3. Wallpaper gets applied and shows up

Would be nice if someone could confirm this to rule out an issue on my side.

**Comment by danyspin97** [comment]
Hello @0xk1f0 and thank you for the report!

Did you build wpaperd in release mode? When it is built in debug mode, the image loading is quite slow and might takes a couple of seconds to load the image.

**Comment by 0xk1f0** [comment]
Yes, I used a modified version of [this](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=wpaperd) PKGBUILD, adapted for 1.0, which builds using `cargo build --frozen --release`. I don't think that's the issue here since the wallpapers almost instantly apply once I issue a command with `wpaperctl`.

**Comment by danyspin97** [comment]
I can reproduce it now, let me fix it right away.

**Comment by danyspin97** [comment]
Fixed by https://github.com/danyspin97/wpaperd/commit/c7d1218d21f7ca12adee8e16856ddd0a76526cd1. I also released 1.0.1 which contain the fix! Let me know if it works now :)

**Comment by 0xk1f0** [comment]
Works now, thanks :)
